### PR TITLE
Changing "Modern Greek" to "Greek", fixes #4070

### LINKED ISSUE #4070: Greek language is listed as "Modern Greek" instead of "Greek"
I was editing a product, and trying to find Greek in the language list, and eventually discovered it in the M's under "Modern Greek", which was not intuitive. 

As we're not likely to enter products in older forms of Greek, perhaps "Greek" could be the preferred term, at least in languages where it changes the prefix?

image image

### PR DESCRIPTION
**Pull request by svensven**
...at least in languages where it changes the sort order.
Translations from https://www.wikidata.org/wiki/Q9129, and any short forms from the infobox on languages' own wikipedia entries for "greek language".

For https://github.com/openfoodfacts/openfoodfacts-server/issues/4070

### CODE REVIEW

File: taxonomies/languages.txt [hunk 1]
```diff
@@ -11324,50 +11324,51 @@ language_code_2:en:mh
...
-de:Neugriechisch, Modernes Griechisch, Neugriechische Sprache
-el:Νέα ελληνική γλώσσα, Νέα ελληνική, Νεοελληνική, Νεοελληνική Γλώσσα, Νέα ελληνικά
+cs:Řečtina, Novořečtina, Moderní řečtina
+da:Græsk, Nygræsk
+de:Griechisch, Griechische Sprache, Neugriechisch, Modernes Griechisch, Neugriechische Sprache
+el:Ελληνικά, Ελληνική γλώσσα, Νέα ελληνική γλώσσα, Νέα ελληνική, Νεοελληνική, Νεοελληνική Γλώσσα, Νέα ελληνικά
 es:griego moderno, Idioma griego moderno
-fi:Nykykreikka
-fo:Nýgrikskt mál
+fi:Kreikan kieli, Nykykreikka
```
Comment (Ban3) [inline taxonomies/languages.txt:11336]:
```suggestion
fi:Kreikka, Kreikan kieli, Nykykreikka
```
Calling it just 'Kreikka' shouldn't be a problem when we know it refers to the language.

File: taxonomies/languages.txt [hunk 2]
```diff
@@ -11324,50 +11324,51 @@ language_code_2:en:mh
...
+cs:Řečtina, Novořečtina, Moderní řečtina
+da:Græsk, Nygræsk
+de:Griechisch, Griechische Sprache, Neugriechisch, Modernes Griechisch, Neugriechische Sprache
+el:Ελληνικά, Ελληνική γλώσσα, Νέα ελληνική γλώσσα, Νέα ελληνική, Νεοελληνική, Νεοελληνική Γλώσσα, Νέα ελληνικά
 es:griego moderno, Idioma griego moderno
-fi:Nykykreikka
-fo:Nýgrikskt mál
+fi:Kreikka, Kreikan kieli, Nykykreikka
+fo:Grikskt mál, Nýgrikskt mál
 fr:grec moderne, Romaïque, el
```
Comment (stephanegigandet) [inline taxonomies/languages.txt:11338]:
```suggestion
fr:Grec, grec moderne, Romaïque, el
```

**Comment by teolemon** [comment]
@Ban3 @stephanegigandet ok to merge ?

**Review by Ban3 — APPROVED** [review]
LGTM

File: taxonomies/languages.txt [hunk 3]
```diff
@@ -11324,50 +11324,51 @@ language_code_2:en:mh
...
 he:יוונית מודרנית
-hi:आधुनिक यूनानी भाषा
-hr:Novogrčki jezik, Novogrčki
-hy:նոր հունարեն
-ilo:Moderno a Griego
+hi:यूनानी, आधुनिक यूनानी भाषा
+hr:Grčki jezik, Novogrčki jezik, Novogrčki
+hy:Հունարեն, նոր հունարեն
+ilo:Griego, Moderno a Griego
 it:greco moderno, Neoellenico, Lingua neogreca, Grammatica greca moderna, lingua greca moderna
```
Comment (VaiTon) [inline taxonomies/languages.txt:11347]:
```suggestion
it:Greco, Lingua greca, Greco moderno, Lingua greca moderna, Neoellenico, Lingua neogreca
```

**Pull request merged**